ui/src/data_explorer/containers/DataExplorer.js

```diff
@@ -79,15 +84,24 @@ const DataExplorer = React.createClass({
       queryConfigs,
       queryConfigActions,
       source,
+      hideWriteFormAction,
+      showWriteFormAction,
+      dataExplorer: {showWriteForm},
     } = this.props
     const {activeQueryIndex} = this.state
 
     return (
       <div className="data-explorer">
+        {showWriteForm
```

Comment:
I'm curious why `showWriteForm` is being handled by Redux state rather than DataExplorer component state?